server/src/test/java/org/cloudfoundry/identity/uaa/oauth/jwt/ChainedSignatureVerifierTests.java

```diff
@@ -88,31 +88,31 @@ public void setup() {
     @Test
     public void test_single_key_valid() {
         verifier = new ChainedSignatureVerifier(new JsonWebKeySet<>(Collections.singletonList(validKey)));
-        signedValidContent.verifySignature(verifier);
+        JwtHelper.decode(signedValidContent.getEncoded()).verifySignature(verifier);
```

Comment:
these corrections are for a better API unterstanding because singing and verification should be done with extra API objects ... before it was with internal state of the objects.... and therefore not really doing a signature check...